src/components/pickers/DatePicker.js

```diff
@@ -19,53 +20,32 @@ export default {
       currentMonth: null,
       currentYear: null,
       isSelected: false,
-      isReversing: false
+      isReversing: false,
+      narrowDays: [],
+      firstDayOffset: 0
     }
   },
 
   props: {
+    locale: {
+      type: String,
+      default: 'en-us'
```

Comment:
When I've previously had to detect local browser locale the solutions have been pretty hacky. But I can take another look.